.bit/responses/2.8-Week 2 Step 8.md

```diff
@@ -1,74 +1,56 @@
----
-files: n/a
-stepType: IssueComment
-scripts: n/a
-week: 2
-step: 8
-name: Week 2 Step 8
----
-
-## Week 2 Step 8
-
-### Checkpoint 2
-
-Time to test our completed Azure Function! It should now successfully do these tasks:
-
-* Parse the image
-* Send image to Face API and return Face data
-
-This time, we won't need to add any additional code, as the completed function should return the emotion data on its own.
-
-<details>
-<summary>These are the same steps as Checkpoint #1. Click here if you need them:</summary>
-<br>
-
-* Navigate back to the Postman Chrome extension app and change GET to POST
-
-![Untitled_ Nov 11, 2020 6_24 PM](https://user-images.githubusercontent.com/69332964/98876201-c3bca780-244b-11eb-9b94-8d3cecc80115.gif)
-
-* Copy your function's url from the Azure Function App portal like this:
-
-![httptrigger - Microsoft Azure](https://user-images.githubusercontent.com/69332964/98876502-6f65f780-244c-11eb-832b-a25888b980da.gif)
-
-* Use the function url and any image you want to send the POST request. Remember to attach the file in Body!
-
-![Untitled_ Nov 11, 2020 6_40 PM](https://user-images.githubusercontent.com/69332964/98876997-780afd80-244d-11eb-87fc-13822d909f2f.gif)
-
-</details>
-
-Only difference is that we should **receive an output in Postman instead:**
-
-Make sure you're using an image with a *real face* on it or else it won't work. Here's an example of an output I get with this image:
-
-![image](https://user-images.githubusercontent.com/69332964/98884689-91687580-245e-11eb-98d7-6461ac79e02a.jpg)
-*Credits: https://thispersondoesnotexist.com/*
-
-```json
-{
-  "result": [
-    {
-      "faceId": "d25465d6-0c38-4417-8466-cabdd908e756",
-      "faceRectangle": {
-        "top": 313,
-        "left": 210,
-        "width": 594,
-        "height": 594
-      },
-      "faceAttributes": {
-        "emotion": {
-          "anger": 0,
-          "contempt": 0,
-          "disgust": 0,
-          "fear": 0,
-          "happiness": 1,
-          "neutral": 0,
-          "sadness": 0,
-          "surprise": 0
-        }
-      }
-    }
-  ]
-}
-```
-
-**To move on, comment the output you received from the POST request!**
+---
```

Comment:
For this page, none of the code markdown works because you have to separate it from the HTML tags. LMK if you need help with that!